### PR TITLE
unset buildopts/installopts before installing Python extensions

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by pforai** [comment]
Looks good, but could be generalized in future for other stuff where we have extensions like R, Perl, etc.

**Comment by boegel** [comment]
I wouldn't generalise it unless it's really needed, and even then I would make it optional by introducing an extra easyconfig parameter to enable/disable it.

For it, only doing it here (where we have a use case for it) should suffice.

**Comment by boegel** [comment]
@wpoely86: thoughts on this one?

**Comment by rjeschmi** [comment]
It looks ok, I just wonder whether it indicates there should be a stronger separation between the Lang EB and it's extensions. Probably something to think about, but this looks fine for now.

File: easybuild/easyblocks/p/python.py [hunk 1]
```diff
@@ -31,7 +31,7 @@
 @author: Pieter De Baets (Ghent University)
 @author: Jens Timmerman (Ghent University)
 """
-
+import copy
```
Comment (wpoely86) [inline easybuild/easyblocks/p/python.py]: keep an empty line here?

Comment (boegel) [inline easybuild/easyblocks/p/python.py]: there's no reason imho

Comment (wpoely86) [inline easybuild/easyblocks/p/python.py]: maybe we really should have a style guide. Some easyblocks have a empty line, some don't.

Comment (boegel) [inline easybuild/easyblocks/p/python.py]: if this is the main thing we're going to worry about, we should consider EB finished ;)

**Comment by wpoely86** [comment]
Looks fine IMHO.

**Comment by boegel** [comment]
Thanks for the review everyone!

File: easybuild/easyblocks/p/python.py (hunk 1 above)
Comment (wpoely86) [inline easybuild/easyblocks/p/python.py]: but but but but, we need something to bitch about! How can we consider this to be a true OSS project if we can't bitch about style?

Comment (boegel) [inline easybuild/easyblocks/p/python.py]: ;-)

**Pull request merged**